test/cli_test.py

```diff
@@ -0,0 +1,299 @@
+# -*- coding: utf-8 -*-
+import inspect
+import io
+import os
+import sys
+import textwrap
+import unittest
+
+# prefer local copy to the one which is installed
+# hack from http://stackoverflow.com/a/6098238/280539
+_top_level_path = os.path.realpath(
+    os.path.abspath(
+        os.path.join(os.path.split(inspect.getfile(inspect.currentframe()))[0], "..")
+    )
+)
```

Comment:
This can be written more succinctly as:
```python
from pathlib import Path
_top_level_path = Path(__file__).parent.parent.absolute()
```
However what you did works, and I'm fine with it.